tests/unit/processor/dissector/test_dissector_rule.py

```diff
@@ -74,8 +74,8 @@ def test_create_from_dict_returns_dissector_rule(self):
                     "filter": "message",
                     "dissector": {"mapping": {"field": "%{ts}"}},
```

Comment:
If this is a valid rule now, shouldn't we test if this rule would lead to the corresponding expected result?

Reply:
I added a test for that :+1: